### DISCUSSION TITLE
introduction

### DISCUSSION THREAD

**Discussion by bilalshaikh292** [life-itself/community#159] [Introductions]
I am Bilal Shaikh,I am a full stack developer,I have experience in web development,software development and also mobile development. I am new to git and github ,I am preparing for my GSOC.I am willing to do all the development work just need a little support.
Thank you

**Comment by rufuspollock** [comment]
@bilalshaikh292 welcome! 👋

As a coder could I suggest that as a first issue you take on something in our associated Flowershow project https://github.com/flowershow/flowershow for example https://github.com/flowershow/flowershow/issues/118
